src/frontend/state/UpdateState.ts

```diff
@@ -0,0 +1,73 @@
+import { makeAutoObservable, runInAction } from 'mobx'
+import { GameInfo } from 'common/types'
+import gameRequiresAccessCodes from 'frontend/helpers/gameRequiresAccessCodes'
+
+class UpdateState {
```

Comment:
Maybe change this to `GameUpdateState`? Update feels a bit too broad to me

Reply:
yup. done